files/certs/ca-key.pem

```diff
@@ -1,51 +1,51 @@
-L3C62kojd1/NxYmVvnm6REU1krWA+ak2HdVbxjHS6bmHUI0CvmMddr6qALpJhk+F
-tMUNYvDGqiRd92hjqq3n/XJCK3u0wXOJxqSQbGdCVFS7DrO/jUEeIlahlDcA7iK8
-QwaY6Ze0R2HTcs/cKGbm+qC/b0naNcPtCqRaEZ06QfEHE1/rOWItsYPyB+N25wAm
-yAIUOBIiM4sNTEPxv17Fg1N+RJFspRAZ6D5NGCUY/h8T6tyqfICDHPhhPZDt+80e
-xULjBSX+OBqzu0E//2sg9wr/6gkpk6+r1zWjm82SEdxmuDOXUPqBwB4obmmXvymk
-SMfLo+EXAQAzZ4+/R1I+GWc2x146MPu/M5ras+PL5HD+ELhr5y2ofL9HbZMCAwEA
-AQKCAgABz1z0YHHVKCMkZPutLNSwIDUD+ro8bQIqLv8ghd6KzF6itFMTY13Ln6xY
-Buj608GWq5vQdy6ANBKt8HKcBMXjHkXs+d3Wesu0xc8zt/t/0lswWQLQTmUJgWzb
-PdfNKsIOAqXT3WriJbZ/GlhevrqBKhnbfw19M3I0wuM1Xb3tqaf9H0b2LS9KYgi9
-cGjhihy2M42rBo+5DDyU+9IqvyFA+PwM9yyMVyMbCvyGWvVEC+3Pq9geBHVTQFIX
-p8rbxS4WivEDDRDIVLc7wXJo9aNBhsV5rS5klnS9KOOe1AwtOYz2pUdMem2L6WJz
-EyUY6rZQVLIx6uHs6C2tsaavvS6rGnVhVo2fVnfHJW0R0GwQk/m+ozTiYuUQB3nd
-LLdHcnoVfORazHK9eQx57VdciYAfdUpoTtQpti0mXt4CPgNePx7MeFJQ2XmUZn9g
-45WHC0T8JYbZYwBDrGMGuPI8SAUKigBFd1CnqMvqTYueD/FyAqXy4BnXGS/9vRwL
-ivpEpgZkJBb7MFWKIyvH3IUnbanCWXHS0HzuPMIVdy6z5Y04j90nHcQCxw86J622
-pFOYjpfUYuJdvicXeUmfrlS8IxIfkRPmYhlwjoGrSUftybpVK+dp+YvF89DSstbk
-HSU1i1UqzcHBUN+4yPr+JAbPpYdvvIgaCKRrNkgO3jutxKSkgQKCAQEAyzuB+O4m
-s0cV7InNXoDWNVyZ4PKNL92q2WZDTf83kil9AL1llNgN+QPq3d6CV2Gez+RRYoB2
-HfvZ++u5p0jNAMdtg8mV+S/5jC06rKRua1aIR2vC+1xdO7d+SrfVUqaOdG5EZQu0
-VeGABhgAWGfTmzgfg/yhcXeqyHDhXV9MTgXW2nhQYYC66yQOQa4s4b37UdhC4jEU
-WiL3vdfKUtsSiwYCg6ebzD2ptmjkhpgK3uNnQZhG+L6Qqsekex77I43mTWEPMLlA
-jcBXxpsa6aUn5YFEQPpgeDOomImTT1upJbJdWgnukJMZsDlgRa6Z+XBbhK+Bk32W
-U7ypqBBAsCRaMwKCAQEAxiwvagiQWw1GRZy/RxPGr5iDiThNAVd4FASCezlYtG8n
-6aekfoE6x/h8VWa5e9AXIJR4riodkAoioL08iiGv9VxF17iPPtWgkg21dpHxybCa
-hFIqHbJKMeMwWlNZOM58A9puaKFoDZPgP5eF3EI9wc7mBnJmuanRf2ke5x+G/beQ
-wbayKL7ny6ipt6rmiwKLx6aADh5Y/gGrtxF+BOPWdAIfrkKw6LlQ8VNZrSbuys2p
-kzWhopvJ9h2zdvSYO2sNXmuXYbjJ1J4BB1m4Vb65349kLKdckTKhFcNtra1oq60f
-TOUIrC+tcPbtkGTppHFikWpIzxaYtmryDIn5Z0X/IQKCAQEAoZEkacbCo4HezxZa
-W0fng7Kjbq7FSYlbGsvus+IyxpMPGnh3elYs/Yt8u3f6fQdRnwpvHPTtcjbXfCxR
-gskgmP7ydHPdM6ZNKICIhdmPIuw/GTB1aSlrwV7GGxZ8k+FvnOZQn1gnvKKyCaYp
-GB8qndqHS4Gawz8UWZdLwtRRMDH6NeKyXGFfuDII13Rln+1OECnyOio0eNgVxin4
-YdhhrH5Rn2KvPDbFZguLqejTs5BrdT1fJr8w+WQPwgXGN+TGczu3PM/5nveLkIuc
-YKT4y77xiQrDiKvg26t7nfpLxXlAN6VvsCYRpQRoId2F8US+rQRt6dhMmzOAWOja
-1G2B+wKCAQEAg1usfht5HHGbH+97trB8Jc7xeZaRLI9/L72hg+6KRufbS/zsg1JD
-1WN3fuBdmJ1hhx4DNGcEO4MMWwG0zFbyHLByN0DF6r+N9OITY6nv3GIsugMAmbih
-h5nxZh4TaR0thRxXzREYOM/JeD2G5G4gI4RyYtmYD/9pt4vFiSwTvb/cuxrP00Rf
-v1ZoUS9+K6yfXNKX1UP2d0BCQ1Hb93/SmgkCWO8p9slHxv8Z1TWWWWWjvslw/6lG
-YNZlLDeLZO9b6DdnG8ZyOnYVdnrZPjTzXiXdH+A78p7dueBVpibo18skkULL0d60
-l8mFxQ2u9WqXDI6n/mUbCueu0/LtF7CFIQKCAQEAh81NQG4bv0x453CgEpoEEK23
-6yPhcVUvE/IkYbgyaWVBAgOFFklSjEZm5P+Jwu1Jbe6OQ8mWhAfGuA4EbH1hXuJD
-4LQ6NrVOQr7jrJ0nrz6WKrbMJucezM03mRh0Rde5GYZ5m0Z/emD63nvfiOZQGfhr
-NAgAtw1k+kYBA4TfycPo6dCKUWUMCoTMhmwr84q6XIJffN/ExO7+osgJPaubOvQf
-ePeuL2W78D3CDMpAjTv/CCQWuTACwHSW3L+KLYdYYf9IlQHDHBPnZbOQgOfVeJzQ
-ds+neHqdc7Zclv1vbk3Mys8PXEyWb/eYi0o9tOgY19Tdb7tcPpYz0WKU2mjKrA==
+MIIJJQIBAAKCAgEAtNdFZzSdHoCsSdQwHdzNnLv0kKD5f0r+6Uu8/CqUrWYHqJqo
```

Comment:
Maybe we should dynamically generate the cert on every machine if they are not there and add the folder to `.gitignore` (remove on `make cleanup`). Github always pushes security alerts for them and when renewing them we have a lot of changed lines.

Reply:
By migrating the shell script into a Makefile?